### ISSUE TITLE
Can't set mpv as the video player

### ISSUE THREAD

**Issue by snow212-cn** [SubtitleEdit/subtitleedit#6487]
The mpv option is always grayed, even when I have copied mpv-2.dll to the same folder where SubtitleEdit.exe is..

**Comment by niksedk** [comment]
Are you on a 32-bit or 64-bit OS?
Windows 11?

**Comment by niksedk** [comment]
Have you tried to download mpv via SE?
Also, what SE version are you using?

![image](https://user-images.githubusercontent.com/1261331/205697503-b91a39c6-72b7-4045-b456-37c64e1065aa.png)

**Comment by hariomsinghchauhan40** [comment]
> Have you tried to download mpv via SE? Also, what SE version are you using?
> 
> Yes I tried it but it shows some "files missing error so I download mpv lib from mpv.io but also after that subtitle edit does not let me set it as mpv .

Subtitle Edit Version - 3.6.10

**Comment by hariomsinghchauhan40** [comment]
> Have you tried to download mpv via SE? 
When ever I tried to down it . This message occured.
![IMG_20221205_224818](https://user-images.githubusercontent.com/119880225/205701202-e3138952-caae-41b0-b239-8e200d5812ce.jpg)

**Comment by hariomsinghchauhan40** [comment]
> Could you attach the "Files missing" message? How did you install SE?

![Uploading 16702608392626883425673901269884.jpg…]()

**Comment by hariomsinghchauhan40** [comment]
I down load SE directly from SE website.

**Comment by niksedk** [comment]
To the SE data folder? Press Ctrl+Alt+Shift+D inside SE to open the SE data folder.

**Comment by hariomsinghchauhan40** [comment]
> To the SE data folder? Press Ctrl+Alt+Shift+D inside SE to open the SE data folder.

Yeah I did it already and all mpv dll which I download are also present on it but SE does not let me downloading mpv.

**Comment by niksedk** [comment]
> > To the SE data folder? Press Ctrl+Alt+Shift+D inside SE to open the SE data folder.
> 
> Yeah I did it already and all mpv dll which I download are also present on it but SE does not let me downloading mpv.

OK, could it be a firewall issue?

**Comment by hariomsinghchauhan40** [comment]
> > > To the SE data folder? Press Ctrl+Alt+Shift+D inside SE to open the SE data folder.
> > 
> > 
> > Yeah I did it already and all mpv dll which I download are also present on it but SE does not let me downloading mpv.
> 
> OK, could it be a firewall issue?

No i don't think .

**Comment by hariomsinghchauhan40** [comment]
> To the SE data folder? Press Ctrl+Alt+Shift+D inside SE to open the SE data folder.

Sir .. very very thanks 😇. I found the problem . You were right .. .Actually i have two folders of name Subtitle Edit.So i downloaded the mpv dll in the wrong Subtitle Edit folder that's why it was not working but when I type ctrl+alt+shift+D then I came to know that I saved downloaded files in wrong folder. And then i send all downloaded file to the right Subtitle Edit folder .and now it is working . Now mpv format is visible in my Subtitle Edit software😀. So now my problem is all solved. Thank you Sir🙏🏻😇Thank you very much. It's all because of you😇🙏🏻

**Comment by JAK1234567** [comment]
> > Have you tried to download mpv via SE? Also, what SE version are you using?
> > Yes I tried it but it shows some "files missing error so I download mpv lib from mpv.io but also after that subtitle edit does not let me set it as mpv .
> 
> Subtitle Edit Version - 3.6.10

same issue for me too. not windows 11, but 10 64 bit

**Comment by Johnson-yue** [comment]
Hi , I have same questeion , I can not download mpv in SE  ,because some network error, so I downloaded file with another way  and got mpv-2.dll , where to place it for SE using..!!!!
